src/main/play-doc/operation/ClusterUpgrade.md

```diff
@@ -0,0 +1,116 @@
+# Upgrading ConductR Cluster
+
+A running cluster can be upgraded without downtime either at the node or cluster level.
+
+On a per node basis, new updated nodes are introduced to the cluster while the old nodes are removed from the cluster. The per node update strategy applies to both ConductR Core node or ConductR Agent node.
+
+On a per-cluster basis, incoming requests are directed from the old cluster to the new cluster using load balancer membership or DNS changes.
+
+## Per node upgrade
+
+Per node upgrades are generally easier. However, they cannot be performed across ConductR releases that are marked as not compatible with previous releases.
+
+To perform a per node upgrade for ConductR Core, introduce new cluster members to the cluster. The new nodes could be created with updated versions of ConductR, Java, the Linux operating system or any other component installed during provisioning. As old members are removed from the cluster, bundles will be replicated to the new resources.
+
+To perform a per node upgrade for ConductR Agent, connect the new agent node to an existing cluster. As old agent nodes are removed from the cluster, bundles will be rescheduled for execution on the new agent nodes.
+
+To perform a per node upgrade for ConductR Core and ConductR Agent installed on the same host, connect both the ConductR Core and ConductR agent to an existing cluster. As old hosts (containing both old ConductR Core and ConductR Agent) are removed from the cluster, bundles will be replicated to the new resources, and bundles will be rescheduled for execution on the new agent nodes.
+
+It is critical to allow sufficient time for bundle relocation and replication before removing old members. In addition to ConductR replicating bundles to new nodes, stateful bundles may require additional time to replicate application data. Removing an agent node before application data has fully replicated can result in application data loss. Elasticsearch bundle provided along with ConductR, for example, requires [verification](#Elasticsearch-Verification) to ensure the data is transferred into the new agent node.
```

Comment:
Changed it to `advised` and `The Elasticsearch bundle`. What additional text do you suggest to perform the `conduct info` check. Shouldn't the user use the HTTP endpoint instead of `conduct info`? Note that this page has not been changed by the PR, it has been just moved to a separate page.